modules/openy_features/openy_node/modules/openy_node_alert/src/Service/AlertManager.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Drupal\openy_node_alert\Service;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+
+/**
+ * Class AlertManager.
+ *
+ * @package Drupal\openy_node_alert\Service
+ */
+class AlertManager {
+
+  /**
+   * The alerts array.
```

Comment:
It is not alerts array - this is alert builders array. Please fix comment and variable names.